<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groupProgressClass</class>
 <widget class="QDialog" name="groupProgressClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>groupProgress</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTreeWidget" name="groupList">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Mark</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Main File</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total Files</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Decoded</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Failed</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Failed Blocks</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cancelled</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>On hold</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>On hold Blocks</string>
       </property>
      </column>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QFrame" name="frame_4">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="dirLabel">
                 <property name="text">
                  <string>Unpack dir: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="repairLabel">
                 <property name="text">
                  <string>Master Repair file: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="unpackLabel">
                 <property name="text">
                  <string>Master Unpack file: </string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QFrame" name="frame_5">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QTreeWidget" name="onHoldList">
                 <property name="autoScroll">
                  <bool>false</bool>
                 </property>
                 <property name="rootIsDecorated">
                  <bool>false</bool>
                 </property>
                 <column>
                  <property name="text">
                   <string notr="true">On hold file subjects</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item>
                <widget class="QTreeWidget" name="decodedList">
                 <property name="autoScroll">
                  <bool>false</bool>
                 </property>
                 <property name="rootIsDecorated">
                  <bool>false</bool>
                 </property>
                 <column>
                  <property name="text">
                   <string notr="true">Decoded files</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Log messages</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPlainTextEdit" name="logWindow">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="markButton">
        <property name="text">
         <string>&amp;Mark all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deletePushButton">
        <property name="text">
         <string>&amp;Delete marked</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshPushButton">
        <property name="text">
         <string>&amp;Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okPushButton">
        <property name="text">
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="quban.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>groupProgressClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1019</x>
     <y>698</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
